content/kubermatic/main/installation/install-kkp-CE/_index.en.md

```diff
@@ -310,35 +381,26 @@ Output will be similar to this:
 ```
 
 If the certificate does not become ready, `kubectl describe` it and follow the chain from Certificate to Order to Challenges.
-Typical faults include bad DNS records or a misconfigured KubermaticConfiguration pointing to a different domain.
+Typical faults include:
 
-### Initializing the first Kubermatic Admin user
+* Bad DNS records.
+* A misconfigured KubermaticConfiguration pointing to a different domain.
 
-With all this in place, you should be able to access https://kubermatic.example.com/ and login either with your static
-password from the `values.yaml` or using any of your chosen connectors. This will initiate your first contact with the 
-KKP API which will create an initial User resource for your account. To become a KKP admin, edit your User instance 
-and set the `admin` flag to `true`. 
+All pods running inside the `kubermatic` namespace should now be running. If they are not, check their logs to find out what's broken.
 
-```yaml
-apiVersion: kubermatic.k8c.io/v1
-kind: User
-metadata:
-  creationTimestamp: "2020-05-04T07:20:37Z"
-  name: 3ec2d8e832964a39b8e8a3df5d25adeea8677f5d6d4706bf23842d3d9663d37c
-spec:
-  admin: true
-  email: example@example.com
-  id: eb279a47c2c869ea89efa48c193ff813b34b96b5c81ada345629205a
-  name: KKP User
-```
+## First Sign In
 
-This will allow you to use the KKP UI and API as an admin. Other users can be promoted to Admins using the [Admin Panel]({{< ref "../../tutorials-howtos/administration/admin-panel/administrators" >}})
+With all this in place, you should be able to access https://kubermatic.example.com/ (i.e. the URL to your KKP setup that you
```

Comment:
Since this URL is not actually working, I'd put it in a code block.

```suggestion
With all this in place, you should be able to access `https://kubermatic.example.com/` (i.e. the URL to your KKP setup that you
```